orchestration/dagster_orchestration/hca_orchestration/resources/bigquery.py

```diff
@@ -8,11 +10,6 @@ def bigquery_client(init_context: InitResourceContext) -> Client:
     return Client(credentials=get_credentials())
 
 
-class NoopBigQueryClient:
-    def create_dataset(self, dataset: Dataset) -> None:
-        pass
-
-
 @resource
 def noop_bigquery_client(init_context: InitResourceContext) -> Client:
-    return NoopBigQueryClient()
+    return Mock(spec=Client)
```

Comment:
I am leaning toward ripping out all of our "noop" resources and replacing them with `ResourceDefinition.hardcoded_resource(Mock(spec=<class of the thing being mocked>))`, as needed. I'd be interested in any thoughts folks have on this.